packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -56,7 +56,7 @@ To use GitHub as the source of a CodePipeline:
 
 ```typescript
 // Read the secret from ParameterStore
-const token = new cdk.SecretParameter(this, 'GitHubToken', { ssmParameter: 'my-github-token' });
+const token = cdk.SecretValue.ssmSecure('my-github-token','1');
```

Comment:
Unfortunately, we have a problem here. CodePipeline does not accept SSM Secure parameters (only Plain Text ones), and trying to deploy with it results in an error like:


```
 ❌  Pipeline failed: ValidationError: SSM Secure reference is not supported in: 
[AWS::CodePipeline::Pipeline/Properties/Stages,AWS::CodePipeline::Webhook/Properties/AuthenticationConfiguration/SecretToken]
```

We need to change this example to use `cdk.SecretValue.secretsManager()`.

Reply:
PR: https://github.com/awslabs/aws-cdk/pull/3113